timeseries/src/autogluon/timeseries/models/chronos/pipeline/utils.py

```diff
@@ -8,6 +11,44 @@
 from autogluon.timeseries.dataset.ts_dataframe import TimeSeriesDataFrame
 
 
+def left_pad_and_stack_1D(tensors: List[torch.Tensor]) -> torch.Tensor:
+    max_len = max(len(c) for c in tensors)
+    padded = []
+    for c in tensors:
+        assert isinstance(c, torch.Tensor)
+        assert c.ndim == 1
+        padding = torch.full(size=(max_len - len(c),), fill_value=torch.nan, device=c.device)
+        padded.append(torch.concat((padding, c), dim=-1))
+    return torch.stack(padded)
+
+
+def download_prefix(bucket, prefix, local_path, force: bool = False, boto3_session=None) -> None:
```

Comment:
Maybe we can use some of the existing functionality in `autogluon.common.loaders.load_s3` (or at least move this method to `common`)?